projects/S905/packages/initramfs/package.mk

```diff
@@ -0,0 +1,71 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2017-present Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="initramfs"
+PKG_VERSION=""
```

Comment:
`PKG_VERSION="1.0"` or something

Reply:
if you look in packages/virtual/initramfs/package.mk there is also no version specified there

Reply:
Its not directly a build problem, if nothing is set you get 0.0-invalid as version. So setting some value is preferred.

Reply:
It hasn't been an issue for me before, nevertheless I have changed it